Web/src/pages/Budget.jsx

```
@@ -146,11 +172,22 @@ function Budget() {
                   {/* I will add self-balancing sliders here for each category */}
-                  <ModalCategoryBudgetSlider  category='Eating Out' />
-                  <ModalCategoryBudgetSlider  category='Groceries' />
+                  {categoryToBudgetDictionary ? (
+                      categoryToBudgetDictionary.map(object => {
+                          const budget = object.budget ? object.budget : 0;
+                          
+                          return (
+                              <ModalCategoryBudgetSlider
+                                  category={object.category}
+                                  budget={budget}
+                              />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)